Sources/Passage/PassageAPIClient/PassageAPIClient.swift

```diff
@@ -432,7 +432,7 @@ internal class PassageAPIClient : PassageAuthAPIClient {
                 
         let (responseData, response) = try await URLSession.shared.upload(for: request, from: data)
 
-        try assertValidResponse(response: response, responseData: responseData)
+        try assertValidResponse(response: response, responseData: responseData, successStatusCode: 201)
```

Comment:
Swift education time. Where are you declaring the non-success state? 

"/// - Throws: ``PassageAPIError``"

Reply:
If I understand your question right - looks like Blayne handled those in the assertValidResponse function, and that error will get thrown all the way up to the dependent app.
https://github.com/passageidentity/passage-ios/blob/621ddc6e3461b0a035d2382f0ae32d540d799452/Sources/Passage/PassageAPIClient/PassageAPIClient.swift#L729

Reply:
Sweet thank you. Do you think there is value in us passing through the success code to the function or can we infer that based on the type of request we are sending?